include/soralog/sink.hpp

```diff
@@ -29,15 +29,23 @@ namespace soralog {
    */
   class Sink {
    public:
+    enum class ThreadFlag {
```

Comment:
I think that a more specific name would be clearer, as it seems to be not just a thread flag, but a thread flag that specifically tells how to identify a thread.

Reply:
Agree. I hope `UsingThreadInfo` is more appropriate name.